src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -7851,6 +7851,42 @@ @using Test
         CompileToAssembly(generated);
     }
 
+    [IntegrationTestFact, WorkItem("https://github.com/dotnet/razor/issues/9592")]
+    public void GenericComponent_TypeParameterOrdering()
```

Comment:
Yes, I thought we don't want to emit baselines unless necessary (because we have so many of them). Here it doesn't seem necessary as the test fails with other errors without the fix.

Reply:
I originally expected other errors to fail and I would update their baselines but it none did, so I guess it would be good to have at least one baseline for this scenario. I will add it.

Reply:
Yeah I agree that we have a lot of baselines, but in this particular case I do agree that we should have the baselines given that the issue is the order and having the generated code allows us to verify that.